### PR TITLE
320 rounds testing: Catchpoint Label Test

### PR DESCRIPTION
**Pull request by nicholasguoalgorand**
## Summary

Test that catchpoint labels are generated (or not generated) under different local configurations.

## Test Plan

This is a test

### CODE REVIEW

File: test/e2e-go/features/catchup/catchpointCatchup_test.go [hunk 1]
```diff
@@ -233,3 +233,124 @@ func TestBasicCatchpointCatchup(t *testing.T) {
...
+		next(response, request)
+	})
+	a.NoError(err)
+	defer wp.Close()
+
+	primaryNodeStatus, err := primaryNodeRestClient.Status()
+	a.NoError(err)
+	a.NotNil(primaryNodeStatus.LastCatchpoint)
+	if expectLabels {
+		a.NotEqual("", *primaryNodeStatus.LastCatchpoint)
```
Comment (tolikzinovyev) [inline test/e2e-go/features/catchup/catchpointCatchup_test.go:297]: I think there is `Empty()` and `NotEmpty()` for this use-case.

File: test/e2e-go/features/catchup/catchpointCatchup_test.go [hunk 2]
```diff
@@ -233,3 +233,124 @@ func TestBasicCatchpointCatchup(t *testing.T) {
...
+			break
+		}
+		currentRound++
+
+	}
+	log.Infof("done building!\n")
+	primaryListeningAddress, err := primaryNode.GetListeningAddress()
+	a.NoError(err)
+
+	wp, err := fixtures.MakeWebProxy(primaryListeningAddress, func(response http.ResponseWriter, request *http.Request, next http.HandlerFunc) {
```
Comment (tolikzinovyev) [inline test/e2e-go/features/catchup/catchpointCatchup_test.go:282]: What does this do?

**Review by tolikzinovyev — COMMENTED** [review]
I'm not familiar with e2e tests, but I left some stylistic comments.

File: test/e2e-go/features/catchup/catchpointCatchup_test.go [hunk 3]
```diff
@@ -233,3 +233,124 @@ func TestBasicCatchpointCatchup(t *testing.T) {
 	secondNode.StopAlgod()
 	primaryNode.StopAlgod()
 }
+
+func checkCatchPointLables(t *testing.T, a *require.Assertions, consensus config.ConsensusProtocols, cfg config.Local, log logging.Logger, expectLabels bool) {
```
[the remark this answers is not in the document]
Comment (nicholasguoalgorand) [reply test/e2e-go/features/catchup/catchpointCatchup_test.go:237]: yes, in initializing fixture

File: test/e2e-go/features/catchup/catchpointCatchup_test.go (hunk 2 above)
Comment (nicholasguoalgorand) [reply test/e2e-go/features/catchup/catchpointCatchup_test.go:282]: didn't need, deleted

**Review by algorandskiy — APPROVED** [review]

**Pull request merged**